server/api/installer.go

```diff
@@ -110,7 +110,10 @@ func (a *Installer) Run() {
 	a.httpServer = &http.Server{}
 	a.httpServer.Addr = fmt.Sprintf("%s:%s", config.Args.Host, config.Args.Port)
 	a.httpServer.Handler = a.router
-	controllers.AddCancelHandler("API Server", a.stopServer)
+	err := controllers.AddCancelHandler("API Server", a.stopServer)
```

Comment:
We are logging the error but still starting the server. Technically the cancel handler isn't needed for the server but if we can't add it, there has to be a reason for it and we should not be ignoring the error.

Reply:
I didn't really think about any of these, just added them, but you make good points so I'll update..  Changed this one to Fatalf.